cmd/tnf/addclaim/addclaim.go

```diff
@@ -27,12 +29,257 @@ var (
 		Short: "The test suite generates a \"claim\" file",
 		RunE:  claimUpdate,
 	}
+	claimCompareFiles = &cobra.Command{
+		Use:   "compare",
+		Short: "Compare 2 \"claim\" file",
+		RunE:  claimCompare,
+	}
 )
 
 const (
 	claimFilePermissions = 0o644
 )
 
+type claimFileStruct struct {
+	Claim struct {
```

Comment:
Why not just use the `Claim` struct directly?  Do we need `claimFileStruct` at all?